### ISSUE TITLE
Hash checks of blackboxes seems to still not work with remote launching

### ISSUE THREAD

**Issue by mseeger** [awslabs/syne-tune#462]
When running an experiment remotely, using simulator backend with `nas201-cifar100` blackbox, this fails with:

```
<html><head></head><body>

Starting experiment (RS/nas201-cifar100/0) of tmlr-3
--
#015  0%\|          \| 0/6 [00:00<?, ?it/s]WARNING:root:Files seem to be corrupted (hash mismatch), regenerating it locally and overwrite files on S3.
start: downloading compressed file
Time for downloading compressed file: 9.1107 secs
start: uncompressing and loading
2022-12-15 21:24:46,707 sagemaker-containers ERROR    Reporting training FAILURE
2022-12-15 21:24:46,707 sagemaker-containers ERROR    framework error:
Traceback (most recent call last):  File "/miniconda3/lib/python3.8/site-packages/sagemaker_containers/_trainer.py", line 84, in train    entrypoint()  File "/miniconda3/lib/python3.8/site-packages/sagemaker_sklearn_container/training.py", line 39, in main    train(environment.Environment())  File "/miniconda3/lib/python3.8/site-packages/sagemaker_sklearn_container/training.py", line 31, in train    entry_point.run(uri=training_environment.module_dir,  File "/miniconda3/lib/python3.8/site-packages/sagemaker_training/entry_point.py", line 99, in run    return runner.get(runner_type, user_entry_point, args, env_vars, extra_opts).run(  File "/miniconda3/lib/python3.8/site-packages/sagemaker_training/process.py", line 286, in run    process = check_error(  File "/miniconda3/lib/python3.8/site-packages/sagemaker_training/process.py", line 204, in check_error    raise error_class(
sagemaker_training.errors.ExecuteUserScriptError: ExecuteUserScriptError:
ExitCode -9
ErrorMessage ""
Command "/miniconda3/bin/python hpo_main.py --experiment_tag tmlr-3 --method RS --num_seeds 1 --save_tuner 0 --scheduler_kwargs grace_period=1,reduction_factor=3 --start_seed 0 --support_checkpointing 1 --verbose 0"
ExecuteUserScriptError:
ExitCode -9
ErrorMessage ""
Command "/miniconda3/bin/python hpo_main.py --experiment_tag tmlr-3 --method RS --num_seeds 1 --save_tuner 0 --scheduler_kwargs grace_period=1,reduction_factor=3 --start_seed 0 --support_checkpointing 1 --verbose 0"

</body></html>
```

**Comment by mseeger** [comment]
When I run the same experiment locally, it all works:

```
Starting experiment (RS/nas201-cifar100/0) of tmlr-3
INFO:root:Skipping download of nasbench201 as /Users/matthis/.blackbox-repository/nasbench201 already exists, change skip_if_present to redownload
INFO:root:Skipping download of nasbench201 as /Users/matthis/.blackbox-repository/nasbench201 already exists, change skip_if_present to redownload
```

**Comment by mseeger** [comment]
Details

Branch: `tmlr_experiments`

Remote launch command:

`python benchmarking/nursery/tmlr_experiments/tmlr3/launch_remote.py --experiment_tag tmlr-3 --num_seeds 1 --method RS`

Local launch command:

`python benchmarking/nursery/tmlr_experiments/tmlr3/hpo_main.py --experiment_tag tmlr-3 --method RS --benchmark nas201-cifar100 --num_seeds 1`

**Comment by mseeger** [comment]
I'll deactivate hash checking for now. But there seem to be two issues:

- Hash signals that BB has changed when run remotely, but not when run locally
- Recreation of BB from scratch fails when run remotely

**Comment by mseeger** [comment]
Related to #476 . It seems the hash computation gives different results on different systems.

**Comment by mseeger** [comment]
In current main, hash checking is switched off by default.

**Comment by mseeger** [comment]
With hash checking deactivated by default, this becomes enhancement.

**Comment by mseeger** [comment]
Closed as duplicate (or special case) of #476
